src/Microsoft.SqlTools.ServiceLayer/Metadata/MetadataService.cs

```diff
@@ -59,7 +59,6 @@ public void InitializeService(ServiceHost serviceHost)
             serviceHost.SetRequestHandler(MetadataListRequest.Type, HandleMetadataListRequest, true);
             serviceHost.SetRequestHandler(TableMetadataRequest.Type, HandleGetTableRequest, true);
             serviceHost.SetRequestHandler(ViewMetadataRequest.Type, HandleGetViewRequest, true);
-            serviceHost.SetRequestHandler(GenerateServerContextualizationRequest.Type, HandleGenerateServerContextualizationNotification, true);
```

Comment:
Wasn't entirely sure if I should keep the original 2 endpoints and add a 3rd one that handles both getting and generating context. I can add it back, but I also don't see a need for 3 endpoints (get context, generate context, get and generate context)

Reply:
Just having the one makes sense to me!